backend/databasemodel/alembic/versions/031ffec68d66/upgrade.sql

```diff
@@ -0,0 +1,28 @@
+ALTER TABLE lipas.abstract ADD COLUMN deleted boolean NOT NULL DEFAULT false;
```

Comment:
Yeah, it all depends what we are going to do with the lipas tables in the first place. Currently they are not used anywhere in the UI, but it was easiest to have the same logic also when updating those tables.

Reply:
If we will have old lipas data in the lipas tables that has been removed in lipas, then it makes it even more difficult to use lipas tables somewhere in my opinion.

Reply:
I see your point. Let's add deleted field for lipas.abstract table too.

Btw. was there a reason you did not, in contrast to all the other tables, create index for lipas.abstract table? If this was not intentional, add
CREATE INDEX ON kooste.abstract (deleted);
on upgrade.sql and model.sql

Reply:
Ah. That is because we don't access the table currently from the UI, so it doesn't have to be indexed to allow fast filtering with the `deleted` field.

Reply:
Also, I don't know what happens if an index is added to a table that is inherited 😅